packages/nimble-components/src/anchor-button/tests/anchor-button.stories.ts

```diff
@@ -35,8 +36,8 @@ const metadata: Meta<AnchorButtonArgs> = {
     parameters: {
         docs: {
             description: {
-                component:
-                    'An anchor button is a component with the visual appearance of a button, but it navigates like an anchor/link when pressed.'
+                component: `An anchor button is a component with the visual appearance of a button, but it navigates like an anchor/link when pressed.
+                    ${targetGuidelines}`
```

Comment:
This is very much usage docs so we should go ahead and add mdx files for these stories and use that pattern.

See the button mdx docs: https://github.com/ni/nimble/blob/main/packages/nimble-components/src/button/tests/button.mdx

Specifically things to note:
- See the pattern for sharing doc fragments from mdx files, i.e. the `ContentHiddenDocs`
- You'll need to remove the autodocs tags: https://github.com/ni/nimble/blob/fde3626559e2469f8540c45d9ff774838b49bb75/packages/nimble-components/src/anchor/tests/anchor.stories.ts#L23